test/smrt/data/feed_dynamic_provider.txt

```diff
@@ -0,0 +1,6 @@
+"64.125.239.15","3cx","tcp,tls","2024-09-14 09:26:56.724585","","provider1.ren-isac.net"
+"122.233.251.205","juniper","tcp,rdp","2024-09-14 08:47:02.129406","","provider2.ren-isac.net"
+"122.233.251.205","sugarcrm","tcp,tls,http","2024-09-14 09:26:55.475147","ja3:d6828e30ab66774a91a96ae93be4ae4c","provider1.ren-isac.net"
+"v6821e39ab66824a91c96ae3be4ae4c","sugarcrm","tcp,tls,http","2024-09-14 09:26:55.475147","ipv4:45.35.20.231","provider2.ren-isac.net"
```

Comment:
"v" isn't a valid character for a hexadecimal hash
Valid characters are 0-9 and a-f